flow/importer/uw/parts/course/import.go

```diff
@@ -20,11 +20,17 @@ func ImportAll(state *state.State, client *api.Client) error {
 		util.CurrentTermId(), util.NextTermId(),
 	}
 	for _, termId := range termIds {
-		term, err := term.Select(state.Db, termId)
+		term_object, err := term.Select(state.Db, termId)
 		if err != nil {
 			log.Warnf("no record for term %04d, proceeding anyway", termId)
+
+			// Create a dummy term object with only the ID set
+			term_object = &term.Term{
+				Id: termId,
```

Comment:
I think start and end date need to be set too

Reply:
Nvm gonna manually create the term object, this doesn't actually create a DB object and we don't have a foreign key constraint on termId for sections so there's no issues on import, only on calendar upload